app.rb

```diff
@@ -1,9 +1,22 @@
-require 'sinatra/base'
+require 'sinatra'
+require 'sinatra/reloader' if development?
+require './lib/peepviewer'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/view_peeps' do
```

Comment:
This is great but the naming is a little unconventional.

I'd expect the routes to be

`GET '/peeps'` for viewing all peeps
`POST '/peeps'` for creating a new peep

And I would expect the model to be called `Peep` so in the controller you'd do things like `Peep.all` and `Peep.create`.

There are also conventions for template names :)

`index` for the page that shows all the peeps

Reply:
Ah! I see that there _is_ a model called `Peep` and that you've implemented another class which takes care of the database interaction. This is a nice idea! I'd probably call that class something like `PeepRepository`